draft-ietf-tls-dtls13.md

```diff
@@ -2016,6 +2083,136 @@ This section provides the normative protocol types and constants definitions.
 %%## ACKs
 %%## Connection ID Management
 
+# Analysis of Limits on CCM Usage {#ccm-bounds}
```

Comment:
General question: Is this analysis something that should be part of the spec, or should it rather be published separately and only referenced here?